app/views/content/is-teaching-right-for-me/maths/maths/_article.html.erb

```diff
@@ -0,0 +1,95 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      If maths is your passion, you'll know that it's not just learning formulas or solving equations. It's a way of thinking and problem solving that’s important in every walk of life.
+    </p>
+
+    <p>
+      An exciting career spent passing your knowledge on to the next generation could be for you.
+    </p>
+
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "quote",
+      colour: "pink",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0024.jpg",
+      image_2_alt: "",
+    ) do %>
+      <p>
+        quote      </p>
+    <% end %>
+  </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <h2>What you'll be teaching</h2>
+      <p>You'll teach the national curriculum. The themes you're likely to cover include: </p>
+      <p>Themes you’ll cover for 11 to 16 year olds (key stage 3 and 4):</p>
```

Comment:
@caitlin-reidy do we have any examples, like for the computing one where we talk about building robot cars, of how this translates to the classroom and what type of practical things they might be doing? For each of the different key stages?

Reply:
No, haven't heard back from maths curriculum team yet.